website/docs/r/lb_server_pool.html.markdown

```diff
@@ -0,0 +1,129 @@
+page_title: "vCloudDirector: vcd_lb_server_pool"
+sidebar_current: "docs-vcd-resource-lb-server-pool"
+description: |-
+  Provides a NSX load balancer server pool data source.
+---
+
+# vcd\_lb\_server\_pool
+
+Provides a vCloud Director Edge Gateway Load Balancer Server Pool data source. A Server Pool defines
+a group of backend servers (defined as pool members), manages load balancer distribution methods, and has a service 
+monitor attached to it for health check parameters.
+
+
+~> **Note:** To make load balancing work one must ensure that load balancing is enabled on edge gateway. This depends 
+on NSX version to work properly. Please refer to [VMware Product Interoperability Matrices](https://www.vmware.com/resources/compatibility/sim/interop_matrix.php#interop&29=&93=) 
+to check supported vCloud director and NSX for vSphere configurations.
+
+~> **Note:** The vCloud Director API for NSX supports a subset of the operations and objects defined in the NSX vSphere 
+API Guide. The API supports NSX 6.2, 6.3, and 6.4.
+
+Supported in provider *v2.4+*
+
+## Example Usage
+
+```hcl
+provider "vcd" {
+  user     = "${var.admin_user}"
+  password = "${var.admin_password}"
+  org      = "System"
+  url      = "https://AcmeVcd/api"
+}
+
+resource "vcd_lb_server_pool" "web-servers" {
+  org                 = "my-org"
+  vdc                 = "my-org-vdc"
+  edge_gateway        = "my-edge-gw"
+
+  name                = "web-servers"
+  description         = "description"
+  algorithm           = "round-robin"
+
+  monitor_id = "${vcd_lb_service_monitor.lb-service-monitor.id}"
+
+  member {
+    condition       = "enabled"
+    name            = "member1"
+    ip_address      = "1.1.1.1"
+    port            = 8443
+    monitor_port    = 9000
+    weight          = 1
+    min_connections = 0
+    max_connections = 100
+  }
+
+  member {
+    condition       = "drain"
+    name            = "member2"
+    ip_address      = "2.2.2.2"
+    port            = 7000
+    monitor_port    = 4000
+    weight          = 2
+    min_connections = 6
+    max_connections = 8
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `edge_gateway` - (Required) The name of the edge gateway on which the server pool is to be created
+* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
+* `name` - (Required) Server Pool name
+* `description` - (Optional) Server Pool description
+* `algorithm` - (Required) Server Pool load balancing method. Can be one of `ip-hash`, `round-robin`, `uri`, `leastconn`, `url`, or `httpheader`
+* `algorithm_parameters` - (Optional) Valid only when `algorithm` is `httpheader` or `url`. The `httpheader` algorithm
+parameter has one option `headerName=<name>` while the `url` algorithm parameter has option `urlParam=<url>`. 
```

Comment:
I would add another section above with a separate "advanced" example, which would use these and other more complex options, but would leave the existing simple sample intact not to scare first time users.

Reply:
Made the first example even less complicated and added a second one with more configuration (and service monitor datasource reference). Both tested with copy/paste so should be working as needed.

Reply:
Let it be data source (two words). Seems this is the valid naming, and mentioned in terraform site. I "optimized" it too much.
Done.